src/main/scripts/src/app/core/rest/organization.service.ts

```diff
@@ -39,15 +38,15 @@ export class OrganizationService {
     return this.httpClient.get<Organization>(OrganizationService.apiPrefix(code));
   }
 
-  public deleteOrganization(code: string): Observable<Response> {
+  public deleteOrganization(code: string): Observable<Object> {
```

Comment:
Isn't it better to use `object` instead of `Object` according to Typescript's "Do's and Don'ts" https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html? :)